trip_app/core/utils.py

```diff
@@ -25,12 +30,29 @@ def str_to_geopoint(data):
         E.g. '23.4 23.5', '[23.4 23.5]', '(23.4 23.5)' would be converted into
         the same object Point(23.4, 23.5) """
     lon, lat = re.findall(r"[-+]?\d*\.?\d+|\d+", data)
```

Comment:
If `re.findall` does not find anything, it returns empty `[]`. So if you try to get `lon` and `lat` it will throw ValueError.

Reply:
Wow. Good point, thanks. Should be updated now.